### ISSUE TITLE
Can Raspberry Pi3 replace Jetson TX2?

### ISSUE THREAD

**Issue by dwkim31** [ROBOTIS-JAPAN-GIT/turtlebot3_slam_3d#1]
I am using intel D435 and raspberry pie, remote pc.

I know Rtabmap works in the following way in Raspberry Pi3.

![image](https://user-images.githubusercontent.com/40584736/45658625-0ebc2800-bb2b-11e8-9148-95c63393f9e0.png)

http://wiki.ros.org/rtabmap_ros/Tutorials/RemoteMapping

http://wiki.ros.org/rtabmap_ros/Tutorials/SetupOnYourRobot

But i think Raspberry Pi3 can't calculate yolo algorithms in this source files.

In this case, What parts of this source need to be modified so that the yolo algorithm can be computed on a Remote pc?

In other words, Is there any problem in compressing and importing the published image from intel d435 and running Detection image parts and Coordinates parts below figure?

![image](https://user-images.githubusercontent.com/40584736/45658749-dcf79100-bb2b-11e8-9954-e731a806a0b5.png)

**Comment by Affonso-Gui** [comment]
Hello,

There is no problem in running parts of the algorithm in a remote PC, which is easily enabled by the ROS framework.

Instead of launching [turtlebot3_slam_3d.launch](https://github.com/ROBOTIS-JAPAN-GIT/turtlebot3_slam_3d/blob/master/launch/turtlebot3_slam_3d.launch,) as a whole, launch each of its nodes separately. You can for instance launch `turtlebot3_d435_bringup.launch` and `rtabmap.launch use_zed:=false rviz:=false` from RasPi and `darknet.launch use_zed:=false` from your remote PC.

It is, however, advisable to have good networking environment and a CUDA-enabled PC.

If YOLO is still too heavy, you can change the model to `yolov2-tiny.yaml` in [darknet_ros.launch](https://github.com/Affonso-Gui/darknet_ros/blob/master/darknet_ros/launch/darknet_ros.launch#L14) or reduce the size in the [yolov2.cfg](https://github.com/Affonso-Gui/darknet_ros/blob/513f93eea10b78f8db0e77bb8df7275eb699faa1/darknet_ros/yolo_network_config/cfg/yolov2.cfg#L8-L9).

By the way are you following [this](https://github.com/IntelRealSense/librealsense/blob/master/doc/RaspberryPi3.md) instructions to run D435 on RasPi? Since there is no USB 3.0 port the image quality seems to be quite low, which may affect the darknet detection accuracy.

**Comment by dwkim31** [comment]
First of all thank you for your kindness reply.

This is a great example of RGB-D SLAM using turtlebot3. so I will try this source with turtlebot3 burger version.

I am planning to make a port using RasPi with the USB 3.0 expansion board to solve the problem you mentioned.

If i have some questions during application, i will ask you again. 

Thank you.
